TransferMethod/Tests/AddTransferMethodPresenterTests.swift

```diff
@@ -8,7 +11,7 @@ class AddTransferMethodPresenterTests: XCTestCase {
     private let mockView = MockAddTransferMethodViewTests()
     private lazy var transferMethodConfigurationFieldsResponse = HyperwalletTestHelper
         .getDataFromJson("TransferMethodConfigurationFieldsResponse")
-    private var mockHyperwalletInsights = MockHyperwalletInsights()
+    private var hyperwalletInsightsMock = HyperwalletInsightsMock()
```

Comment:
rename MockAddTransferMethodViewTests to AddTransferMethodViewMock

Reply:
fixed